guides/faq.rst

```diff
@@ -259,9 +259,12 @@ And a docker compose file looks like this:
         image: esphome/esphome
         volumes:
           - ./:/config:rw
+          - /etc/localtime:/etc/localtime:ro
         network_mode: host
         restart: always
 
+To ensure the timezone of the ESPHome container is correct, only map `/etc/localtime`; ESPHome has no need for `/etc/timezone` nor a `TZ` environment variable.
```

Comment:
Remove this line - the user really shouldn't care *what* file are included in general docker, the docker compose example already shows how it's done for ESPHome.